src/pages/ClerkScreen.tsx

```diff
@@ -39,15 +54,50 @@ const ClerkScreen = ({
   toggleLiveMode,
   unconfigure,
 }: Props) => {
-  const changeAppPrecinctId: EventTargetFunction = (event) => {
-    const currentTarget = event.currentTarget as HTMLInputElement
-    const appPrecinctId = currentTarget.value
-    setAppPrecinctId(appPrecinctId)
+  const changeAppPrecinctId: SelectChangeEventFunction = (event) => {
+    setAppPrecinctId(event.currentTarget.value)
   }
 
   const [isTestDeck, setIsTestDeck] = useState(false)
   const showTestDeck = () => setIsTestDeck(true)
   const hideTestDeck = () => setIsTestDeck(false)
+
+  const [isDateModalActive, setIsDateModalActive] = useState(false)
+  const [systemDate, setSystemDate] = useState(new Date())
+  const [systemTimeZone, setSystemTimeZone] = useState('America/Chicago')
+  const updateSystemDate: InputChangeEventFunction = (event) => {
+    const [year, month, day] = event.currentTarget.value.split('-')
+    setSystemDate(
+      new Date(
+        parseInt(year),
+        parseInt(month) - 1,
+        parseInt(day),
+        systemDate.getHours(),
+        systemDate.getMinutes()
+      )
+    )
+  }
+  const updateSystemTime: InputChangeEventFunction = (event) => {
+    const [hours, minutes] = event.currentTarget.value.split(':')
+    setSystemDate(
+      new Date(
+        systemDate.getFullYear(),
+        systemDate.getMonth(),
+        systemDate.getDay(),
+        parseInt(hours),
+        parseInt(minutes)
+      )
```

Comment:
```suggestion
      new Date(
        systemDate.getFullYear(),
        systemDate.getMonth(),
        systemDate.getDate(),
        parseInt(hours, 10),
        parseInt(minutes, 10)
      )
```

`Date#getDay()` returns the day of the week, not the day of the month.